.gitpod/.gitpod.Dockerfile

```
@@ -8,11 +8,11 @@ RUN sudo apt-get -qq install -y python3 python3-pip libxext6 libxrender1 libxtst
 RUN pip3 install projector-installer
 # Install PhpStorm
 RUN mkdir -p ~/.projector/configs  # Prevents projector install from asking for the license acceptance
-RUN projector install 'PhpStorm 2021.1' --no-auto-run
+RUN projector install 'PhpStorm 2021.2.2' --no-auto-run
```

```suggestion
RUN projector install 'PhpStorm 2020.3.3' --no-auto-run
```
When running `projector install`, it asks `Do you want to select from Projector-tested IDE only?`
If yes - this list is displayed:
```
PhpStorm 2019.3.4
PhpStorm 2020.2
PhpStorm 2020.3.2
PhpStorm 2020.3.3
```

We should actually downgrade to `2020.3.3`, because it's a version that was tested in projector.

---

Any reason to install older version? 2021.3 in EAP yet

---

@andypost I'm not sure I understand your last comment.
I was saying we should not install latest version, because they are more buggy when running through Projector.